src/transferslot.cpp

```diff
@@ -433,6 +433,20 @@ void TransferSlot::doio(MegaClient* client)
                 continue;
             }
 
+            if (reqs[i]->status == REQ_FAILURE && reqs[i]->httpstatus == 200 && transfer->type == GET && transferbuf.isRaid())
```

Comment:
Can you leave a comment as to why `200` is to be checked for here?

Reply:
done